R/score.R

```diff
@@ -82,6 +82,14 @@
 #'   add_coverage(by = c("model", "target_type")) %>%
 #'   summarise_scores(by = c("model", "target_type"))
 #'
+#' # set forecast unit and check forecasts in workflow
```

Comment:
add a comment about why you might want to do this

Reply:
made it more verbose